cron/poll_read_issue_comments.py

```diff
@@ -216,28 +241,70 @@ def handle_comment(api, issue_comment):
                                                                            comment=comment_text))
 
             if command == "/vote":
-                handle_vote_command(api, parsed_comment, issue_id,
-                                    global_comment_id, votes)
+                handle_vote_command(api, parsed_comment, issue_id, global_comment_id, votes)
         elif can_run and not has_votes:
             # oops we didn't pass
-            update_db(global_comment_id, {"has_run": True})
-            db_fields = select_db(global_comment_id, ("chaos_response_id", "command"))
+            db = update_db(global_comment_id, {"has_ran": True})
+            db_fields = select_db(global_comment_id, ("chaos_response_id", "command"), db=db)
             resp_id = db_fields["chaos_response_id"]
             command = db_fields["command"]
             body = "> {command}\n\nVote failed".format(command=command)
             gh.comments.edit_comment(api, settings.URN, resp_id, body)
-    else:
-        __log.debug("comment: {comment} is not a command, " +
-                    "parsed: {parsed}".format(comment=comment_text, parsed=orig_parsed))
 
 
+def is_command(comment):
+    comment = re.sub('\s+', ' ', comment)
+    parsed_comment = list(map(lambda x: x.lower(), comment.split(' ')))
+    is_command = parsed_comment[0] in COMMAND_LIST # Command will be at front
+    print("IS_COMMAND" + str(is_command))
```

Comment:
if you're going to be logging some debug commands, use the `_log.debug` instead of print.